src/components/security-code-error/security-code-error-routes.ts

```diff
@@ -35,4 +36,10 @@ router.get(
   securityCodeEnteredExceededGet
 );
 
+router.get(
```

Comment:
maybe this belongs in an mfa-reset component? then it can be hidden behind the switch

Reply:
or in the account-recovery component?